### PR TITLE
chore:[META-993] Add customer facing doc to help users connect to the db

### PR DESCRIPTION
**Pull request by Pivotal-Jeff-Jun**
# Which other branches do you want a technical writer to cherry-pick this PR to (if any)?

It's best to PR to the most recent relevant branch and leave all the cherry-picking to the
writer, except where earlier branches require factual changes to the content.
For more information about the branches, see https://github.com/pivotal/docs-tap#branches

### CODE REVIEW

File: scst-store/connect-to-database.hbs.md [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+    ```shell
+    db_password=$(kubectl get secret postgres-db-password -n metadata-store -o json | jq -r '.data.DB_PASSWORD' | base64 -d)
+    ```
+1. In a separate terminal, run the following command:
+
+    ```shell
+    kubectl port-forward service/metadata-store-db 5432:5432 -n metadata-store
+    ```
+    
+    With this, database host and port values will be the following:
```
Comment (ohlookadollar) [inline scst-store/connect-to-database.hbs.md]: "Set the database host and port values on the first terminal with the following"

**Review by julia-pu — APPROVED** [review]

**Review by ohlookadollar — APPROVED** [review]

File: scst-store/connect-to-database.hbs.md [hunk 2]
```diff
@@ -0,0 +1,43 @@
+# Connecting to the Postgres Database
+
+To connected to the Postgres Database, you will need the following values:
```
Comment (xtreme-debbie-chen) [inline scst-store/connect-to-database.hbs.md]: `connected` -> `connect`

**Comment by xtreme-debbie-chen** [comment]
Should our official documentation recommend connecting to the database with the certificate? 

I know when I wrote the dev docs that it was excluded, but I was mostly being lazy and/or true to how I actually connect to the database myself.

**Comment by Pivotal-Jeff-Jun** [comment]
>

**Comment by Pivotal-Jeff-Jun** [comment]
> Should our official documentation recommend connecting to the database with the certificate?

Yeah I was thinking about that, but to do that, we would need to not use port-forwarding no?
And to not use port-forwarding, don't we need a service with external ip?

**Comment by ohlookadollar** [comment]
> > Should our official documentation recommend connecting to the database with the certificate?
> 
> Yeah I was thinking about that, but to do that, we would need to not use port-forwarding no? And to not use port-forwarding, don't we need a service with external ip?

Good point, yea we should probably put something about connection via tls. So we shouldn't expose the db with external ip, port forwarding should be the only way to access it.

A full verification (ssl-mode verify-full) won't work unless you modify your /etc/hosts. It good enough to use verify-ca because we are using self sign certificate.

**Review by xtreme-debbie-chen — APPROVED** [review]

**Review by ohlookadollar — APPROVED** [review]

**Pull request merged**